src/LibraryManager.Contracts/FileHelpers.cs

```diff
@@ -245,21 +254,33 @@ public static bool DeleteFiles(IEnumerable<string> filePaths)
         /// Deletes folders if empty
         /// </summary>
         /// <param name="folderPaths"></param>
+        /// <param name="rootDirectory"></param>
         /// <returns></returns>
-        public static bool DeleteEmptyFoldersFromDisk(IEnumerable<string> folderPaths)
+        public static bool DeleteEmptyFoldersFromDisk(IEnumerable<string> folderPaths, string rootDirectory)
         {
+            if (folderPaths.Count() == 0)
+            {
+                return true;
+            }
+
+            LocalPathComparer comparer = new LocalPathComparer();
```

Comment:
>comparer [](start = 30, length = 8)

If you make the change above to ensure all of these are already under rootDirectory, then this comparer isn't needed.

Reply:
It's deleted.

---
In reply to: [201166856](https://github.com/aspnet/LibraryManager/pull/286#discussion_r201166856) [](ancestors = 201166856)